### PR TITLE
Fix downvote button style

### PR DESCRIPTION
**Pull request by louisa-dev**
Fix downvote button style from:
FROM downvote button
To:
TO downvote button

Tested on desktop web Firefox

### CODE REVIEW

**Comment by AmitBokoDev** [comment]
@louisa-dev thanks again!

**Comment by AmitBokoDev** [comment]
@louisa-dev I reverted the commit, after your change the downvotes don't show at all on mobile, mobile and desktop use completly different classes and DOM structures. The mobile version is what this line is for-
` $("#"+art_id+" .post-vote").append(`<span class="post-vote__text downvote">${downvotes}</span>`);`

File: assets/js/content.js [hunk 1]
```diff
@@ -156,9 +156,11 @@ async function showDonwvotes(art_id,json){
     }                                   
   }
   ////console.log('postId ',post_id," downvotes", downvotes);
-  if(downvotes!== null){
-    $("#"+art_id+" .post-vote").append(`<span class="post-vote__text downvote">${downvotes}</span>`);
```
Comment (AmitBokoDev) [inline assets/js/content.js:160]: no mention of this line in the new code. This line is for mobile, the next is for desktop (the use different DOM and classes). Without this, downvotes will show only on desktop not on mobile.

**Comment by marcello-dev** [comment]
> @louisa-dev I reverted the commit, after your change the downvotes don't show at all on mobile, mobile and desktop use completly different classes and DOM structures. The mobile version is what this line is for- ` $("#"+art_id+" .post-vote").append(`${downvotes}`);`

I see, I only tested on Firefox desktop and it seems to work. For the next time, how do you test to make sure it works on all platforms?

**Comment by louisa-dev** [comment]
> @louisa-dev I reverted the commit, after your change the downvotes don't show at all on mobile, mobile and desktop use completly different classes and DOM structures. The mobile version is what this line is for- ` $("#"+art_id+" .post-vote").append(`${downvotes}`);`

I see, I only tested on Firefox desktop and it seems to work. For the next time, how do you test to make sure it works on all platforms?

**Comment by AmitBokoDev** [comment]
@louisa-dev the two main platform I care about is chrom/chromium browsers (I use Brave for example), and Firefox. Safari can go to hell for all I care. 
To test it on mobile, in the browser debugger (F12)  use the mobile view
on chrome it's the button that looks like a laptop and phone right next to the inspect button
image
on firefox it looks like a tablet and a phone, left of the 3 dot menu
image

**Comment by louisa-dev** [comment]
Got it, thanks

**Comment by AmitBokoDev** [comment]
@louisa-dev I just want to say I'm not trying to be an asshole by not accepting your commit, it's a good fix it's just that not all of the details are there. If you want to fix it and create another pull request I'll be glad to approve it, if you don't care enough about it I'll still try adjusting and implementing it myself because it's still a good fix.
Thanks for contributing anyway

**Pull request merged**